lib/src/phy/rf/rf_soapy_imp.c

```diff
@@ -298,7 +298,24 @@ int rf_soapy_open_multi(char* args, void** h, uint32_t num_requested_channels)
     printf("\n");
   }
 
-  SoapySDRDevice* sdr = SoapySDRDevice_make(&(soapy_args[0]));
+  // Select Soapy device by id
+  const char dev_arg[]   = "soapy=";
```

Comment:
why don't we just call the param `id=`? seems more intuitive

Reply:
please run clang-format here, the spaces between `=` are too many

Reply:
Fixed.